Rakefile

```diff
@@ -17,7 +17,7 @@ Cucumber::Rake::Task.new(:acceptance) do |t|
     a.push('--color')
     a.push('--format progress')
     a.push('--strict')
-    a.push('--tags ~@wip')
+    a.push('--tags "not @wip"')
```

Comment:
Fixed this so that we don't show this WARN message anymore:
```
Deprecated: Found tags option '~@wip'. Support for '~@tag' will be removed from the next release of Cucumber. Please use 'not @tag' instead.
```